pkg/kubelet/apis/config/fuzzer/fuzzer.go

```diff
@@ -50,7 +50,7 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 			obj.EventBurst = 10
 			obj.EventRecordQPS = 5
 			obj.EnableControllerAttachDetach = true
-			obj.EnableDebuggingHandlers = true
+			obj.EnableDebuggingHandlers = false
```

Comment:
Why do you do that? I don't think debugging handlers really add visible overhead...